### PR TITLE
Converted proxy to installconfig-proxy

### PR DESCRIPTION
**Pull request by degville**
- Added new web GUI image for external proxy server
- Added new web GUI image for enabling/disabling proxy server
- Converted text with small edits for readability (needs more)
- Checked command syntax and output

### CODE REVIEW

File: src/en/installconfig-proxy.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+When you install a Region Controller, maas-proxy will be installed. The
+configuration of the proxy relies on maas-proxy being on the same machine as
+the Region Controller.
+
+### Using an external proxy
+
+If you want to use an external proxy, you can define its URL using the web
+interface by selecting Settings>General and scrolling down to the 'Proxy for
+APT and HTTP/HTTPS' section:
```
Comment (pmatulis) [inline src/en/installconfig-proxy.md]: The proxy is only for APT and not also for HTTP/HTTPS. I've been meaning to open a bug on this for a while. Do we   want to clarify this here?

**Comment by pmatulis** [comment]
All good except for a question. Feel free to ignore and just merge.

**Comment by degville** [comment]
Thanks Peter. I think it is worth clarifying in the text. I'll add some words and merge.

**Comment by degville** [comment]
As per feedback, added clarification on APT. Merging...

**Pull request merged**